federated_search/amp-pd-service/app/database.py

```diff
@@ -0,0 +1,8 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+# Our sqlite database will be a local file named test.db
+DATABASE_URL = "sqlite:///./test.db"
```

Comment:
even for a toy, you might find it pays off to make this configurable now. just my 2c!

Reply:
Yea, thats not a bad idea. How would you suggest I make this configurable?

Reply:
if there isn't some more standard way built into fastAPI, I would probably use an environment variable. You could set `test.db` as the default. e.g. something like `os.getenv("FEDSEARCH_DATABASE_URL", "./test.db")`